packages/x-components/src/x-modules/history-queries/store/__tests__/actions.spec.ts

```diff
@@ -369,9 +401,8 @@ describe('testing history queries module actions', () => {
     });
 
     // eslint-disable-next-line max-len
-    it('updates a history query if search response change because a filter is selected', async () => {
+    it('updates a history query when the search response changes because a filter is selected', async () => {
```

Comment:
It's worth testing with facets with `__unknown__` as id. Create a new test for it and where done 👍